devices.js

```diff
@@ -14205,6 +14205,46 @@ const devices = [
             await reporting.instantaneousDemand(endpoint);
         },
     },
+    {
+        zigbeeModel: ['1GBatteryDimmer50AU'],
+        model: 'AU-A1ZBR1GW',
+        vendor: 'Aurora Lighting',
+        description: 'AOne one gang wireless battery rotary dimmer',
+        fromZigbee: [fz.battery, fz.command_on, fz.command_off, fz.command_step, fz.command_step_color_temperature],
+        toZigbee: [tz.light_brightness_step],
+        exposes: [e.battery(), e.action([
+            'on', 'off', 'brightness_step_up', 'brightness_step_down', 'color_temperature_step_up', 'color_temperature_step_down'])],
+        meta: {configureKey: 1, battery: {voltageToPercentage: '3V_2100'}},
+        endpoint: (device) => {
```

Comment:
This shouldn't be needed.

Reply:
which line are you referring to

Reply:
the device shows a greenpower: -

![image](https://user-images.githubusercontent.com/1891623/103803042-ec29e200-5047-11eb-85c1-7cb6d752e807.png)

not sure if it is of any relevance

Reply:
also 
![image](https://user-images.githubusercontent.com/1891623/103803150-18ddf980-5048-11eb-92f1-5770ff65899d.png)

Reply:
when used with an aurora hub which I do have at hand dimmer ready with a zigbee sniffer also supports changing colour and color wheel possibly scenes

Reply:
I'm refering to the endpiont line

Reply:
I realised it won't be needed as there are only two endpoints1 and 242 and the one being used is 1 any idea what 242 would do (greenpower?)

Reply:
Probably this device can serve as a green power sink or proxy. (for more info search Zigbee green power).